src/cowboy_req.erl

```diff
@@ -1222,26 +1222,43 @@ response_merge_headers(Headers, RespHeaders, DefaultHeaders) ->
 		merge_headers(Headers2, RespHeaders),
 		DefaultHeaders).
 
+camelize_header({N,V}) ->
```

Comment:
Matching at function argument is more idiomatic than one at case expression in erlang if possible.
```erlang
camelize_header({<<"ETag">> = N,V}) ->
    {N, V};
camelize_header({<<"x-amz-meta">> = N,V}) ->
    {N, V};
camelize_header({N,V}) when is_binary(N) ->
    {cowboy_bstr:capitalize_token(N), V};
camelize_header({N,V}) ->
    {N, V}.
```